platform/linux-generic/arch/x86/cpu_flags.c

```diff
@@ -1,8 +1,6 @@
-/* Copyright (c) 2017-2018, Linaro Limited
- * Copyright (c) 2023, Nokia
- * All rights reserved.
- *
- * SPDX-License-Identifier:     BSD-3-Clause
+/* SPDX-License-Identifier: BSD-3-Clause
+ * Copyright (c) 2017-2018 Linaro Limited
+ * Copyright (c) 2023 Nokia
  */
 
 /* SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Intel copyright could be moved to the list above as well.

Reply:
It could, but is there a general convention what should be done in this kind of cases? At least one motivation for keeping the licenses separate is that it's easier to notice that the file contains copied code.

Reply:
Combined copyright messages and listed DPDK source files in V2.